### PR TITLE
WRP-3062: Increased code coverage for Dropdown and Picker related modules

### PR DESCRIPTION
**Pull request by daniel-stoian-lgp**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [x] Documentation was verified or is not changed
* [x] UI test was passed or is not needed
* [x] Screenshot test was verified or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Increased code coverage for Dropdown and Picker related modules

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRP-3062

### Comments

Enact-DCO-1.0-Signed-off-by: Daniel Stoian [daniel.stoian@lgepartner.com](mailto:daniel.stoian@lgepartner.com)

### CODE REVIEW

File: Dropdown/tests/Dropdown-specs.js [hunk 1]
```diff
@@ -1,8 +1,10 @@
 import {FloatingLayerDecorator} from '@enact/ui/FloatingLayer';
 import '@testing-library/jest-dom';
 import {render, screen} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
 
 import Dropdown, {DropdownBase} from '../Dropdown';
+import DropdownList from "../DropdownList";
```
Comment (stanca-pop-lgp) [inline Dropdown/tests/Dropdown-specs.js:7]: Please change to single quotes.

File: TimePicker/tests/TimePicker-specs.js [hunk 2]
```diff
@@ -83,4 +137,36 @@ describe('TimePicker', () => {
...
+
+		expect(firstMeridiemDisplayed).not.toBeNull();
+		expect(secondMeridiemDisplayed).not.toBeNull();
+	});
+
+	describe('#timeToLocaleString', () => {
+		test('method should convert time to a localized string', () => {
+			ilib.setLocale('en-US');
+			const date = new Date(2000, 0, 1);
+			const expected = '12:00 AM';
```
Comment (stanca-pop-lgp) [inline TimePicker/tests/TimePicker-specs.js:160]: Please leave an empty line before expected.

File: DatePicker/tests/DatePicker-specs.js [hunk 3]
```diff
@@ -2,13 +2,30 @@ import '@testing-library/jest-dom';
...
+import DatePicker, {dateToLocaleString} from '../DatePicker';
 
 // Note: Tests pass 'locale' because there's no I18nDecorator to provide a value via context and
 // otherwise, nothing renders in the label.
 
 describe('DatePicker', () => {
-	test('should emit an onChange event when changing a component picker', () => {
+	test('should set the value to the current date when no value is provided', () => {
+		render(<DatePicker locale="en-US" />);
+
```
Comment (stanca-pop-lgp) [inline DatePicker/tests/DatePicker-specs.js:13]: Please remove empty line.

File: DatePicker/tests/DatePicker-specs.js [hunk 4]
```diff
@@ -55,4 +90,20 @@ describe('DatePicker', () => {
 
 		expect(yearPicker).toHaveTextContent(expected);
 	});
+
+	describe('#dateToLocaleString', () => {
+		test('method should convert date to a localized string', () => {
+			const date = new Date(2000, 0, 1);
+			const expected = 'Saturday, January 1, 2000';
```
Comment (stanca-pop-lgp) [inline DatePicker/tests/DatePicker-specs.js:97]: Please leave an empty line before expected.

**Review by stanca-pop-lgp — APPROVED** [review]
LGTM

**Review by adrian-cocoara-lgp — APPROVED** [review]

File: DatePicker/tests/DatePicker-specs.js [hunk 5]
```diff
@@ -55,4 +89,21 @@ describe('DatePicker', () => {
...
+		test('method should convert date to a localized string', () => {
+			const date = new Date(2000, 0, 1);
+
+			const expected = 'Saturday, January 1, 2000';
+			const actual = dateToLocaleString(date);
+
+			expect(actual).toBe(expected);
+		});
+
+		test('method should return \'null\' for an invalid date', () => {
```
Comment (sjro) [inline DatePicker/tests/DatePicker-specs.js:103]:
```suggestion
		test('method should return `null` for an invalid date', () => {
```

File: TimePicker/tests/TimePicker-specs.js [hunk 6]
```diff
@@ -83,4 +137,37 @@ describe('TimePicker', () => {
...
+			ilib.setLocale('en-US');
+			const date = new Date(2000, 0, 1);
+
+			const expected = '12:00 AM';
+			const actual = timeToLocaleString(date);
+
+			expect(actual).toBe(expected);
+		});
+
+		test('method should return \'null\' for an invalid time', () => {
```
Comment (sjro) [inline TimePicker/tests/TimePicker-specs.js:167]:
```suggestion
		test('method should return `null` for an invalid time', () => {
```

**Pull request merged**